styles/about_me.css

```diff
@@ -0,0 +1,55 @@
+
```

Comment:
Re: Chris's comment on drying up your CSS... you could have one file that uses styles you use across all pages (header, nav, some others...) and load that CSS file on each page.  Then, about_me.css, hoby_blog.css etc. would contain the styles that were unique to just that page.  Basically anything that it looks like you copied and pasted into more than one file, I would refactor out into it's own file to be loaded on each page.

You could also add more than one class to elements and apply styling to that.  For example, just from looking at these top two files, .about-me, .gaming, and .hiking look like they apply the same styles (I'm guessing this holds true on other pages, too).  So, you could either do:

```
.about-me, .gaming, .hiking {
shared styles;
}
```

OR

```
.generic-class-name {
shared styles;
}
```

I lean more towards doing the second thing.  You could name is something like .main-page-content (Idk, naming stuff is hard!): http://stackoverflow.com/questions/1790455/whats-the-best-way-to-name-ids-and-classes-in-css-and-html